gui/Test/src/main/java/sample/test/service/MenuItemService.java

```diff
@@ -0,0 +1,106 @@
+package sample.test.service;
+
+import com.google.gson.Gson;
+import sample.test.dto.MenuItemDto;
+import sample.test.helpers.MenuItemResponse;
+import sample.test.model.Category;
+import sample.test.model.MenuItem;
+import sample.test.utils.HttpUtils;
+
+import java.net.http.HttpResponse;
+import java.util.Arrays;
+import java.util.List;
+
+public class MenuItemService {
+
+    public static List<MenuItem> getMenuItems() {
+        try {
+            String url = "http://localhost:8080/api/menu/";
+            String token = JwtTokenService.getInstance().getJwtToken();
+            HttpResponse<String> response = HttpUtils.sendHttpRequest(url, "GET", token, null);
+            String responseBody = response.body();
+            MenuItem[] menuItemsArray = new Gson().fromJson(responseBody, MenuItem[].class);
+            return Arrays.asList(menuItemsArray);
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    public static MenuItem getFirstMenuItem() {
+        List<MenuItem> menuItems = getMenuItems();
+        if (menuItems != null && !menuItems.isEmpty()) {
+            return menuItems.get(0);
+        }
+        return null;
+    }
+
+    public static MenuItem getMenuItemById(Long id) {
+        try {
+            String url = "http://localhost:8080/api/menu/" + id;
+            String token = JwtTokenService.getInstance().getJwtToken();
+            HttpResponse<String> response = HttpUtils.sendHttpRequest(url, "GET", token, null);
+            String responseBody = response.body();
+            MenuItemResponse menuItemResponse = new Gson().fromJson(responseBody, MenuItemResponse.class);
+            return menuItemResponse.getData();
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    public static boolean addMenuItem(MenuItemDto menuItemDto) {
+        try {
+            String url = "http://localhost:8080/api/menu/";
+            String token = JwtTokenService.getInstance().getJwtToken();
```

Comment:
this propably too should be extracted, because it doesnt fit with addMenuItem, (this token getters)